frontend/src/pages/AboutUs/TrackerTab.tsx

```diff
@@ -0,0 +1,201 @@
+import React from "react";
+import Grid from "@material-ui/core/Grid";
+import Typography from "@material-ui/core/Typography";
+import styles from "./AboutUsPage.module.scss";
+import Button from "@material-ui/core/Button";
+import { EXPLORE_DATA_PAGE_LINK } from "../../utils/urlutils";
+
+function TrackerTab() {
+  const OUR_JOURNEY = (
+    <Grid container xs={12} className={styles.GreyGridItem}>
+      <Grid
+        container
+        xs={12}
+        justify="center"
+        className={styles.Grid}
+        spacing={5}
+      >
+        <Grid item xs={6}>
+          <h1>Our journey</h1>
+          <p>
+            Started in 2020 during the COVID-19 pandemic,  the Health Equity
+            Tracker aggregates up-to-date demographic data from communities that
+            are  being hit hardest by the pandemic.
+          </p>
+          <p>
+            The data will power the Health Equity Tracker that’ll give a
+            detailed breakdown of the virus’s impact by race, ethnicity, gender,
+            socioeconomic status, and other critical factors. This tracker will
+            help policymakers better understand what resources and support
+            affected communities need to close gaps and battle the virus
+            effectively.
+          </p>
+        </Grid>
+        <Grid item xs={6}>
+          We’ve made great progress in creating this dashboard:
+          <br />
+          <br />
+          <Grid container spacing={5} justify="center">
+            <Grid item xs={6}>
+              <span className={styles.UnderlinedHeader}>5 data sources</span>
+              <p>
+                HET currently aggregates data from 5 key data sources. We plan
```

Comment:
Stuff like this and the # of variables below seems like it will very likely go out of date. Mind adding a TODO to get the numbers by reading from the dataset metadata and MetricConfig?